code/clue.js

```diff
@@ -1,59 +1,227 @@
+dumbbell,
+poison,
+axe,
+bat,
+trophy,
+pistol
 ]
 
-const weapons = []
+//console.log(weapons[weapons.length - 1]);
+
+
+
 
-const rooms = []
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
 // YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
+
+
 const randomSelector = array => {
-  return array[Math.floor(Math.random() * array.length)]
+return array[Math.floor(Math.random() * array.length)]
 }
+//console.log(randomSelector(suspects));
+
 
 // CREATE AN OBJECT THAT KEEPS THE MYSTERY.
+const mystery = {
+  killer : "",
+  weapons : "",
+  room : ""
+};
+
 // With a killer, a weapon and a room.
 // The values will be set later.
 
 // This function will be invoked when you click on the killer card.
+
 const pickKiller = () => {
   // This will randomly select a killer from the suspects. And add that to the mystery object.
   mystery.killer = randomSelector(suspects)
+  
+  //This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
+  document.getElementById('killerCard').style.background = mystery.killer.color;
+  document.getElementById('killerImage').src = mystery.killer.image;
+  document.getElementById('killerName').innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+}
+
+const pickWeapon = () => {
+  // This will randomly select a weapon. And add that to the mystery object.
+  mystery.weapons = randomSelector(weapons)
+
+  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
+  document.getElementById('weaponCard').style.background = mystery.weapons.color
+  document.getElementById('weaponName').innerHTML = `${mystery.weapons.name} <br> Weight:${mystery.weapons.weight}`
+}
+const pickRoom = () => {
+  // This will randomly select a killer from the suspects. And add that to the mystery object.
+  mystery.room = randomSelector(rooms)
 
   // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
-  document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('roomCard').style.background = mystery.room.color
+  document.getElementById('roomName').innerHTML = `${mystery.room}`
 }
 
-// CREATE FUNCTIONS pickWeapon and pickRoom in a similar way.
 
 // STEP 4 - CREATE A FUNCTION revealMystery that will be invoked when you click that button. It should show something like:
-// 'The murder was committed by Jacob Green, in the living room with a rope.'
+const theMystery = document.getElementById('mystery');
```

Comment:
A variable name describing what kind of element this is could be useful? For example:
`mysteryDiv`
`mysteryButton`
`mysteryHeader`